util/util.go

```diff
@@ -311,6 +311,7 @@ func NormalizeName(name string) (string, string) {
 		}
 	}
 
+	name = filepath.ToSlash(name)
 	root := GetRootFromPackage(name)
```

Comment:
The issue is that name contains backslashes.

With `github.com\foo\bar\baz` as input:

```
// name == "github.com\foo\bar\baz"
root := GetRootFromPackage(name) // root == "github.com/foo/bar"
extra := strings.TrimPrefix(name, root) // extra == "github.com\foo\bar\baz"
```